### PR TITLE
Handle when pdfinfo chokes.

### PR DESCRIPTION
**Pull request by et**
pdfinfo cannot handle a password protected file. Hence, the error for this would get swallowed
since this only goes to stderr. Now it is rejected so it can be handled by the consumer.

### CODE REVIEW

File: test/pdfinfo.test.js [hunk 1]
```diff
@@ -52,13 +52,22 @@ describe('pdfinfo', function(){
   });
 
 
-  describe('error', function(done){
```
Comment (et) [inline test/pdfinfo.test.js]: I'm not even sure if this even worked before since I thought `done`s has to be stuck onto `it` blocks only.

Comment (iapain) [reply test/pdfinfo.test.js]: @et Thanks for catching it :+1:

**Comment by iapain** [comment]
@et Thanks for PR. I am about to merge it. Please note that by sending a PR you agree to MIT license of your contribution. If you have any objection please let me know I will revert it.

**Comment by et** [comment]
Thanks @iapain. Totally fine with the MIT license.
Could you also publish a new release so I can pull the changes into my project?

**Comment by iapain** [comment]
@et published in v0.5.2. Thanks again for PR

**Pull request merged**